gnome/libgtop/Portfile

```diff
@@ -4,36 +4,40 @@ PortSystem          1.0
 PortGroup           gobject_introspection 1.0
 
 name                libgtop
-version             2.36.0
-revision            1
+version             2.40.0
+revision            0
 license             GPL-2
 set branch          [join [lrange [split ${version} .] 0 1] .]
 
 description         A library for portably obtaining information about processes
 
-long_description    libgtop is a library for portably obtaining information about processes, \
-                    such as their PID, memory usage, etc.
+long_description    ${name} is a library for portably obtaining information \
+                    about processes, such as their PID, memory usage, etc.
 
 maintainers         {devans @dbevans} openmaintainer
 categories          gnome
-platforms           darwin
-homepage            https://www.gnome.org/
+homepage            https://www.gnome.org
 master_sites        gnome:sources/${name}/${branch}/
 
 use_xz              yes
 
-checksums           rmd160  f3380ce02650080f0391c16fe07bc46e87dd4a9f \
-                    sha256  13bfe34c150b2b00b03df4732e8c7ccfae09ab15897ee4f4ebf0d16b0f3ba12b
+checksums           rmd160  fc8ce3f9f95de6ede71f8838e3fdddedd3792768 \
+                    sha256  78f3274c0c79c434c03655c1b35edf7b95ec0421430897fb1345a98a265ed2d4 \
+                    size    744316
 
-depends_build       port:pkgconfig \
-                    port:gtk-doc
+depends_build-append \
+                    port:gettext \
+                    port:gtk-doc \
+                    path:bin/pkgconfig:pkgconfig
 
-depends_lib         path:lib/pkgconfig/glib-2.0.pc:glib2
+depends_lib         port:gettext-runtime \
+                    path:lib/pkgconfig/glib-2.0.pc:glib2 \
+                    port:xorg-libXau
 
-patchfiles          patch-libgtop-sysdeps.m4.diff \
-                    patch-add-null-pointer-checker.diff
+patchfiles          patch-libgtop-sysdeps.m4.diff
 
-gobject_introspection yes
+gobject_introspection \
+                    yes
```

Comment:
I'm not sure splitting this into two lines makes things more readable but I don't care that much either way.

Reply:
I could have dropped this, of course.